client/src/components/menubar/authButtons.js

```diff
@@ -1,31 +1,122 @@
-import React from "react";
-import { AnchorButton, ButtonGroup, Tooltip } from "@blueprintjs/core";
+/* eslint-disable camelcase -- Allowing snake_case from BE */
+import React, { useState } from "react";
+import {
+  AnchorButton,
+  ButtonGroup,
+  Elevation,
+  Tooltip,
+  Popover,
+  PopoverPosition,
+  Checkbox,
+  Card,
+  Button,
+  Classes,
+} from "@blueprintjs/core";
 import * as globals from "../../globals";
 import styles from "./menubar.css";
 
+import { storageGet, storageSet, KEYS } from "../util/localStorage";
+
+const LOGIN_PROMPT_OFF = "off";
+
 const Auth = React.memo((props) => {
+  const [isPromptOpen, setIsPromptOpen] = useState(shouldShowPrompt());
+
   const { auth, userinfo } = props;
 
-  if (!auth || (auth && !auth.requires_client_login)) return null;
+  if (shouldShowAuth()) return null;
+
+  const AuthButton = () => {
+    return (
+      <AnchorButton
+        type="button"
+        data-testid="auth-button"
+        disabled={false}
+        href={!userinfo?.is_authenticated ? auth.login : auth.logout}
+      >
+        {!userinfo?.is_authenticated ? "Log In" : "Log Out"}
+      </AnchorButton>
+    );
+  };
 
   return (
     <ButtonGroup className={styles.menubarButton}>
-      <Tooltip
-        content="Log in to cellxgene"
-        position="bottom"
-        hoverOpenDelay={globals.tooltipHoverOpenDelay}
-      >
-        <AnchorButton
-          type="button"
-          data-testid="auth-button"
-          disabled={false}
-          href={!userinfo.is_authenticated ? auth.login : auth.logout}
+      <div style={{ position: "relative" }}>
```

Comment:
Add a relative parent to wrap siblings `<AuthButton />` and `<div style={{ position: "absolute" }} />`, so the "absolute" sibling can overlap with `<AuthButton />` and position `<Popover />` correctly

We can't just wrap Popover > Tooltip > AuthButton, because that disables `<AuthButton />, which means clicking on the `<AuthButton />` will only trigger the `<Popover />` instead of navigating to the login/logout link

Reply:
> which means clicking on the ``will only trigger the`` instead of navigating to the login/logout link

 typo :)

Reply:
Ah thank you! Missing the closing ` 😆 Corrected!